zookeeper-contrib/zookeeper-contrib-monitoring/check_zookeeper.py

```diff
@@ -261,62 +256,21 @@ def _parse_stat(self, data):
                 result['zk_znode_count'] = int(m.group(1))
                 continue
 
-            m = re.match('Watch count: (\d+)', line)
```

Comment:
@cbueche can you elaborate a bit on why do you remove these lines?

Reply:
My patch was against `master` and this code wasn't present in that branch.

Reply:
so apologies I should have checked this. Basically there are two changes in my PR:
-  used 2to3 to make it work with Python 3.x
- loop to get all data in _send_cmd()

Reply:
I can approve the part of 

* used 2to3 to make it work with Python 3.x
* loop to get all data in _send_cmd()

However, in the edge master branch, these matches are still [there](https://github.com/apache/zookeeper/blob/14b936201b719ab5cf6346d1ed7dfc4ff382cbb7/zookeeper-contrib/zookeeper-contrib-monitoring/check_zookeeper.py#L226-L303). So could you explain why you remove these lines?

Reply:
I can't. Probably an error when doing the other changes, I can't explain why I didn't see the "massive" deletion. If you want me to redo a PR, let me know, and against which commit id.